manifests/overlays/prod/secrets/clusters/api.aasthana.dev.datahub.redhat.com.enc.yaml

```diff
@@ -0,0 +1,41 @@
+apiVersion: v1
+kind: Secret
+metadata:
+    name: cluster-apps.ocp.prod.psi.redhat.com
```

Comment:
This will overwrite the ocp4 cluster spec, can you change the name to match your dev cluster